service/sensors/v1/sensors.pb.gw.go

```diff
@@ -15,7 +15,7 @@ import (
 
 	"github.com/grpc-ecosystem/grpc-gateway/v2/runtime"
 	"github.com/grpc-ecosystem/grpc-gateway/v2/utilities"
-	"go.viam.com/api/common/v1"
```

Comment:
why did this change?

Reply:
Not really sure, but I will confirm. I do note multiple PRs over the past few days have switched it to v`1_0` to `v1` or vica versa